src/services/scan_form_service.js

```diff
@@ -12,9 +12,14 @@ export default (easypostClient) =>
      * @returns {ScanForm}
      */
     static async create(params) {
-      // TODO: We should re-implement the check here that wraps up params in `shipments` if the user didn't
       const url = `${this._url}`;
 
+      // wraps up params in `shipments` if the user didn't do it
+      if (!params.shipments) {
+        // eslint-disable-next-line no-param-reassign
+        params = { shipments: params };
+      }
```

Comment:
This isn't quite what I had in mind, the implementation before would pull out individual shipment IDs and assign them each to an `id` key (needed for the API). Can we reimplement that?

Reply:
Resolved